README.md

```diff
@@ -162,6 +162,32 @@ setVariantEmoji(yourClassThatImplementsVariantEmoji)
 
 If no instance or a null instance is set the [default implementation](./emoji/src/main/java/com/vanniktech/emoji/VariantEmojiManager.java) will be used.
 
+## Animattions
+
+### Custom keyboard enter and exit animations
```

Comment:
Why is this here?

Reply:
Where would you put it? It belongs to animations?

Reply:
Ah, it's because like I previously said - this branch builds on the previous one. It'll disappear from the PR once the other PR is merged.

Reply:
Yeah I realized that once I saw the third PR.